README.md

```diff
@@ -93,6 +93,31 @@ the Dagger component, e.g.
 interface AppComponent
 ```
 
+## Contributed bindings
+
+The `@ContributesBinding` annotation generate a Dagger binding method for an annotated class and 
+contributes this binding method to the given scope. Imagine this example:
+```
+interface Authenticator
+
+class RealAuthenticator @Inject constructor() : Authenticator
+
+@Module
+@ContributesTo(AppScope::class)
+abstract class AuthenticatorModule {
+  @Binds abstract fun bindRealAuthenticator(authenticator: RealAuthenticator): Authenticator
+}
+```
+This is a lot of boilerplate if you always want to use `RealAuthenticator` when injecting
```

Comment:
Nice 💯

Reply:
![200](https://user-images.githubusercontent.com/101754/89595038-5bd14200-d808-11ea-90a2-3c24c9bc5278.gif)

Reply:
Holy shit. Have you tweeted this snippet out? This is the code I've always wanted to write. It's so beautiful…

Reply:
Will tweet later today when it's out :)